Wikipedia/Code/WMFAnnouncementsContentSource.m

```diff
@@ -67,6 +67,7 @@ - (void)loadContentForDate:(NSDate *)date inManagedObjectContext:(NSManagedObjec
 
```

Comment:
**Note:** this fix is not in this file, but I just needed to leave a comment somewhere that I can resolve later. This is a bug I missed in the UI PR.

I think the date formatter for the cell subtitle label might be off. It's not noticeable when the section title says "Today", but once the card is on screen, if you terminate and move your device date up a few days, then relaunch and and scroll down until you see the card again, you'll notice that it's off from the section title:

![IMG_0660](https://user-images.githubusercontent.com/3620196/161123778-34535b15-5335-459c-b788-0cee2cffd704.PNG)

Can you update it so it matches with however the section title is calculating its full date string? I think it's [this line](https://github.com/wikimedia/wikipedia-ios/blob/40291b2d8b28d4cb35537c93675e1ad3d2972e6d/Wikipedia/Code/WMFContentGroup%2BWMFFeedContentDisplaying.m#L104).